tests/ImageSharp.Interactive.Tests/FormatterTests.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Threading.Tasks;
+using AngleSharp.Html.Parser;
+using FluentAssertions;
```

Comment:
I really don't like FluentAssertations. They're syntactic sugar I find makes parsing less easy for non-native English speakers.

Reply:
I find it easy and more flexible, but I am happy to comply